src/modality-agnostic-files/data-summary-files.md

```diff
@@ -132,6 +132,177 @@ It is RECOMMENDED to accompany each `samples.tsv` file with a sidecar
 }
 ```
 
+## Phenotypic and assessment data
```

Comment:
This section has been moved to its own file: src/modality-agnostic-files/phenotypic-and-assessment-data.md

Reply:
That's bizarre... I thought I already noticed that and took care of it... Sorry. I'll do another commit addressing just this.

Reply:
Addressed with commit #5718888